scripts/test-migrations.sh

```diff
@@ -298,65 +306,75 @@ pip install -U uv
 # Start the database
 start_db
 
-for VERSION in "${VERSIONS[@]}"
-do
-    test_upgrade_to_version "$VERSION"
-done
+if [ "$MIGRATION_TYPE" == "full" ]; then
+    for VERSION in "${VERSIONS[@]}"
+    do
+        test_upgrade_to_version "$VERSION"
+    done
 
-# Test the most recent migration with MySQL
-test_upgrade_to_version "current"
+    # Test the most recent migration with MySQL
+    test_upgrade_to_version "current"
+else
+    # Test the most recent migration with MySQL
+    test_upgrade_to_version "current"
 
+    # Start fresh again for this part
+    rm -rf "$ZENML_CONFIG_PATH"
 
-# Start fresh again for this part
-rm -rf "$ZENML_CONFIG_PATH"
+    # Fresh database for sequential testing
+    stop_db
+    start_db
 
-# fresh database for sequential testing
-stop_db
-start_db
+    # Test sequential migrations across multiple versions
```

Comment:
```suggestion
    # Test random migrations across multiple versions
```

And please change it so it no longer refers to 'sequential migrations' anywhere else in the script